src/App.js

```diff
@@ -167,62 +162,55 @@ export default class App extends React.Component<{}, AppState> {
                     </Row>
-                    <Row className='App__program-block-editor'>
-                        <Col>
+                    <Row className='App__program-section'>
+                        <Col md={4} lg={3}>
+                            <div className='App__command-palette'>
+                                <div className='App__command-palette-heading'>
+                                    <FormattedMessage id='CommandPalette.movementsTitle' />
+                                </div>
+                                <div className='App__command-palette-command'>
+                                    <CommandPaletteCommand
+                                        commandName='forward'
+                                        icon={React.createElement(
+                                            ArrowForward,
+                                            {className:'command-block-svg'}
+                                        )}
+                                        selectedCommandName={this.getSelectedCommandName()}
+                                        onChange={this.handleCommandFromCommandPalette}/>
+                                </div>
+                                <div className='App__command-palette-command'>
+                                    <CommandPaletteCommand
+                                        commandName='right'
+                                        icon={React.createElement(
+                                            ArrowTurnRight,
+                                            {className:'command-block-svg'}
+                                        )}
+                                        selectedCommandName={this.getSelectedCommandName()}
+                                        onChange={this.handleCommandFromCommandPalette}/>
+                                </div>
+                                <div className='App__command-palette-command'>
+                                    <CommandPaletteCommand
+                                        commandName='left'
+                                        icon={React.createElement(
+                                            ArrowTurnLeft,
+                                            {className:'command-block-svg'}
+                                        )}
+                                        selectedCommandName={this.getSelectedCommandName()}
+                                        onChange={this.handleCommandFromCommandPalette}/>
+                                </div>
+                            </div>
+                        </Col>
+                        <Col md={8} lg={9}>
                             <ProgramBlockEditor
                                 minVisibleSteps={6}
                                 program={this.state.program}
                                 selectedAction={this.state.selectedAction}
+                                runButtonDisabled={this.state.dashConnectionStatus}
```

Comment:
Replace with:

    runButtonDisabled={this.state.dashConnectionStatus !== 'connected'}